packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.types.ts

```diff
@@ -1,16 +1,8 @@
 import { IColumn } from './DetailsList.types';
 import { IDetailsRowStyles, ICellStyleProps } from './DetailsRow.types';
-import { IBaseProps, IRefObject } from '../../Utilities';
 import { IDetailsListProps } from './DetailsList';
 
-export interface IDetailsRowFields {}
```

Comment:
> Just to be clear, we aren't removing IDetailsRowField anywhere, we're just removing componentRef prop from IDetailsRowFieldProps.

@JasonGore It looks like `IDetailsRowFields` is being removed

Reply:
Sorry, this component interface. Yes, it's removed, but wasn't being exported and didn't have any functions defined for it. I suspect this just came from the create-component script per my other comment.

Reply:
Resolving given the above comment kept open.